plane/source/docs/automatic-landing.rst

```diff
@@ -41,13 +41,20 @@ Setting Up the Approach Waypoint
 
+If you are having trouble getting your plane to line up for landing because it seems to be making very wide turns, consider changing `WP_LOITER_RAD <WP_LOITER_RAD>`, if your plane is capable of safely making tighter turns. 
+
 In addition, the last navigation waypoint before NAV_LAND must be far enough away that any turns made by the mission to that waypoint will have already be completed and the vehicle back on track in a line to the landing point. Usually, two waypoints are used. The first to allow any mission item (including aborted go-arounds for an autoland reattempt) to get aligned to another waypoint setup closer to the touchdown point but still providing a nice glide slope. We will designate the last waypoint before the NAV_LAND as the "final approach" waypoint and the one before it the "pre-approach" waypoint, as shown above. The next figure shows these waypoints **incorrectly** placed, without sufficient spacing:
 
 .. image:: ../../../images/autolanding-approach.png
     :target: ../../_images/autolanding-approach.png
 
 The above shows a planned approach with the pre-approach waypoint 440m away from touchdown at 50m altitude, the final approach waypoint at 220m away and 25m altitude, giving a supposedly gentle glide slope of 6 degrees (which is a 1.7m/s descent at 15m/s airspeed). However, the actual result is 9 deg glide path since the switch to AUTO mode and the landing sequence was begun at the point shown at beginning of the yellow path, and the resulting overshoots at the two waypoints result in a steeper glide slope (9 degrees) between the final approach waypoint and the touchdown point. Spacing the waypoints a little further away would have allowed the approach to have settled by the final approach waypoint, and with the desired final glide slope. The faster the vehicle is flying, the wider the turn radii are, and the wider the waypoints should be spaced.
 
+.. note::
+
+   If you use QGroundControl to add add your landing, the waypoints added are slightly differently. When you use the "Land" button, on the Plan screen, QGroundControl will add a "Loiter (altitude)" waypoint (NAV_LOITER_TO_ALT), then a do land start (DO_LAND_START), and finally a NAV_LAND. The loiter to altitude is another way to line up the plane for landing, but if you use this approach, make sure to set the loiter direction correctly so that the plane will exit the loiter lined up correctly for the landing.
```

Comment:
QGC automatically aligns the approach depending on the loiter direction...you can change it, move the waypoint slightly and it will re-align everything, so this warning is not needed....since this is not a QGC wiki, and we don't control it, I would tend to minimize the details....my suggested changes and grammar fixes:
```suggestion
   You can use QGroundControl to automatically setup your approach and landing waypoints, but it will use a slightly different approach. Using the "LAND" button on the Plan screen, QGroundControl will add a "LOITER_TO_ALT" waypoint (NAV_LOITER_TO_ALT) at a distance determined by its wizard's glide slope parameter to bring the vehicle down (or up) to approach altitude, add a DO_LAND_START (See :ref:`aborting autolanding`) marker, and a NAV_LAND.
```

Reply:
Thanks @Hwurzburg - your suggestion is much better.

Reply:
> @timtuxworth do you want me to fix this and merge it since you are using the web editor?

Thanks @Hwurzburg - I'll try to get a local editor working soon.